src/gurt/tests/SConscript

```diff
@@ -0,0 +1,74 @@
+# (C) Copyright 2020 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+# The Government's rights to use, modify, reproduce, release, perform, display,
+# or disclose this software are subject to the terms of the Apache License as
+# provided in Contract No. 8F-30005.
+# Any reproduction of computer software, computer software documentation, or
+# portions thereof marked with this legend must also reproduce the markings.
+"""Unit tests"""
+
+import os
+import daos_build
+
+TEST_SRC = ['test_gurt.c']
+LIBPATH = [Dir('../')]
+
+def scons():
+    """Scons function"""
+    if GetOption('help'):
+        return
+
+    # pylint: disable=too-many-locals
+    Import('env', 'prereqs', 'gurt_targets')
+    # pylint: enable=too-many-locals
+
+    #Use full path to wrap_cmocka.h for configure test.  Since standalone
+    #cmocka header can't be included without including other headers,
+    #this test can't be generalized
+    wrap_cmocka = os.path.join(Dir('.').srcnode().abspath, 'wrap_cmocka.h')
+    prereqs.define('cmockawrap', headers=[wrap_cmocka], libs=['cmocka'],
+                   package='libcmocka-devel')
+    if not prereqs.check_component('cmocka'):
+        print("\n***************************************************")
+        print("libcmocka-devel package needed to enable unit tests")
+        print("***************************************************\n")
+        # Just returning is the desired behavior here.  The utest target
+        # has yet to be defined so the build will just fail.
+        return
+
+    test_env = env.Clone()
+    prereqs.require(test_env, "mercury", "uuid", "cmocka")
+    test_env.AppendUnique(LIBS=['pthread'])
+    test_env.AppendUnique(CPPPATH=['../include'])
```

Comment:
same comment as above

Reply:
Same here. LIBPATH, CPPPATH and RPATH are already part of env.